data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
       "Янсен",
       "Янси",
       "Ян",
-      "Ярбро"
+      "Ярбро",
+      [ "Авель", "Abel" ],
+      [ "Авраам", "Abraham" ],
```

Comment:
This and the one above (Abel), I'm not sure about. On the one hand, the two are famously-biblical names and had been solidifed as such. On the other hand, these are meant to be names as they are used at present, aimed to reflect the reality of New England as it is (or as it would be in this fictional reality; which is to say: very close).

On this basis, I would transcribe them: `Эйбл` (so pronounced like the word `able`) and `Абрахам`. One famous example of that is Dr. Maslow, the famous psychologist behind Maslow's hierarchy of needs: his name in Russian would be [Абрахам Маслоу](https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D1%81%D0%BB%D0%BE%D1%83,_%D0%90%D0%B1%D1%80%D0%B0%D1%85%D0%B0%D0%BC).